demos/using-dev-build/webapp/index.ts

```diff
@@ -61,7 +61,7 @@ document.getElementById("loginBtn")?.addEventListener("click", async () => {
   // At this point we're authenticated, and we can get the identity from the auth client:
   const identity = authClient.getIdentity();
   // Using the identity obtained from the auth client, we can create an agent to interact with the IC.
-  const agent = new HttpAgent({ identity });
+  const agent = await HttpAgent.create({ identity, shouldFetchRootKey: true });
```

Comment:
Always `shouldFetchRootKey`?

Reply:
Yes, this is in the dev build demo, which runs on the local replica and hence always needs fetching the root key.

Reply:
Additional context: there was no `fetchRootKey` before in the example. However, that only worked because the `whoami` call was a query call, which is uncertified.

Anyone expanding that example with an update call would immediately run into an issue. Hence the `fetchRootKey` is added now.